include/pmacc/kernel/warp.hpp

```diff
@@ -53,66 +53,70 @@ namespace pmacc
 
 
 #    if(__CUDA_ARCH__ >= 300 || BOOST_COMP_HIP)
-        /** broadcast data within a warp
-         *
-         * required PTX ISA >=3.0
```

Comment:
Was this requirement dropped?

Reply:
All supported CUDA architectures are 3.5 anyway and even 3.5 is not supported by CUDA 12.X anymore.
So no need to mention it because it is supported for all GPUs (NVIDIA and AMD)